apps/dashboard/src/pages/_document.tsx

```diff
@@ -10,8 +10,6 @@ class ConsoleDocument extends Document {
           {/* preconnect to domains we know we'll be using */}
           <link rel="preconnect" href="https://a.thirdweb.com" />
           <link rel="dns-prefetch" href="https://a.thirdweb.com" />
-          <link rel="preconnect" href="https://pg.paper.xyz" />
```

Comment:
also remove it from `CSP` in next.config.js

Reply:
removed, good catch